dev/sparktestsupport/modules.py

```diff
@@ -389,18 +389,6 @@ def __hash__(self):
         "python/pyspark/sql"
     ],
     python_test_goals=[
```

Comment:
This seems like it was included by mistake? If you did this to quickly run selected tests you can use the `--module` with `./python/run-tests`